pkg/network/vmispec/interface.go

```diff
@@ -50,14 +50,27 @@ func FilterInterfacesSpec(ifaces []v1.Interface, predicate func(i v1.Interface)
 }
 
 func IsPodNetworkWithMasqueradeBindingInterface(networks []v1.Network, ifaces []v1.Interface) bool {
-	if podNetwork := lookupPodNetwork(networks); podNetwork != nil {
+	if podNetwork := LookupPodNetwork(networks); podNetwork != nil {
 		if podInterface := LookupInterfaceByNetwork(ifaces, podNetwork); podInterface != nil {
 			return podInterface.Masquerade != nil
 		}
 	}
 	return true
 }
 
+func PopInterfaceByNetwork(statusIfaces []v1.VirtualMachineInstanceNetworkInterface, network *v1.Network) (*v1.VirtualMachineInstanceNetworkInterface, []v1.VirtualMachineInstanceNetworkInterface) {
+	if network == nil {
+		return nil, statusIfaces
+	}
+	for index, currStatusIface := range statusIfaces {
```

Comment:
Aint it gosec G601 ?
See please https://github.com/kubevirt/kubevirt/pull/4751

Reply:
fixed